.github/workflows/test.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         restore-keys: venv-${{ runner.os }}-${{ steps.ubuntu-version.outputs.UBUNTU_VERSION }}-${{ matrix.python-version }}-
 
     - name: Install developer tools
-      run: make bootstrap
+      run: pip install .
```

Comment:
What is the difference between this and the previous command?

Reply:
The previous command installed digitalmarketplace-developer-tools from PyPI; this install digitalmarketplace-developer-tools from this repo.